src/client/components/Terms.tsx

```diff
@@ -42,50 +37,51 @@ const TermsLink = ({
 	</ExternalLink>
 );
 
-export const GuardianTerms = (props: TermsProps) => (
-	<div css={termsBox(props)}>
-		<Text>
-			By proceeding, you agree to our{' '}
-			<TermsLink href="https://www.theguardian.com/help/terms-of-service">
-				terms &amp; conditions
-			</TermsLink>
-			. For information about how we use your data, see our{' '}
-			<TermsLink href="https://www.theguardian.com/help/privacy-policy">
-				privacy policy
-			</TermsLink>
-			.
-		</Text>
-	</div>
+export const TermsBox = ({
```

Comment:
In the new layout, we're going to be using this element for stuff that isn't terms, like here for instance:

![Screenshot 2024-02-28 at 15 47 19](https://github.com/guardian/gateway/assets/101555242/b7d03a4f-ece4-4074-954d-beb073d514c5)

Can we call it something more generic, like `AsideBox`?

Reply:
Refactored and renamed into `InformationBox`